controllers/grid/submissions/SubmissionsListGridCellProvider.inc.php

```diff
@@ -104,9 +136,25 @@ function getTemplateVarsFromRowColumn($row, $column) {
 		$context = $contextDao->getById($contextId);
 
 		switch ($columnId) {
+			case 'id':
+				return array('label' => $submission->getId());
 			case 'title':
-				return array('label' => '');
-				break;
+				$this->_titleColumn = $column;
+				$title = $submission->getLocalizedTitle();
+				if ( empty($title) ) $title = __('common.untitled');
+
+					$primaryAuthor = $submission->getPrimaryAuthor();
+					$authors = $submission->getAuthors();
+					if (!isset($primaryAuthor)) {
+						if (sizeof($authors) > 0) {
+							$primaryAuthor = $authors[0];
+						}
+					}
+
+				$authorsInTitle = $primaryAuthor->getLastName();
+				if (count($authors) > 1) $authorsInTitle = $authorsInTitle . ' et al.';
```

Comment:
(...plus IMO we should do this in a standard way throughout the system, like getAuthorString does. If we need to cases -- a long and a short author string, where the short one uses "et al" -- then that should be implemented in the same place as getAuthorString.)